pkgs/tools/security/vault/vault-bin.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation rec {
+  name = "vault-${version}";
```

Comment:
Can you please change name to `vault-bin-${version}`, if you query profie packages bi name, it will be easier to see which vault have you installed.